### ISSUE TITLE
Error: 400 - Only upto 2 sort fields are allowed.

### ISSUE THREAD

**Issue by waako** [typesense/typesense-instantsearch-adapter#32]
## Description

Getting a console error with results not rendering when using `sort_by` with 3 values.  
According to https://typesense.org/docs/0.17.0/api/ up to 3 sort fields can be specified but that throws an error in `TypesenseInstantsearchAdapter.js`.  
We are using typesense server 0.19.0

## Steps to reproduce

Configure sort to use 3 fields:  
```js
<SortBy
    defaultRefinement={`${indexName}/sort/poa:asc,model_weight:desc,price:asc`}
    items={[
        {label: 'Price (Lowest)', value: `${indexName}/sort/poa:asc,model_weight:desc,price:asc`},
        {label: 'Price (Highest)', value: `${indexName}/sort/poa:asc,model_weight:desc,price:desc`},
        …
    ]}
/>
```

## Expected Behavior

No example URL, but would expect results to be returned with the appropriate sort order

## Actual Behavior

0 results, with console error(s):  
```
Error: 400 - Only up to 2 sort fields are allowed. TypesenseInstantsearchAdapter.js:76
```

## Metadata

**Typsense Version**: `1.0.0`

**OS**:  macOs 10.15.7

**Comment by jasonbosco** [comment]
@waako This was fixed in the public release of v0.19. Related: https://github.com/typesense/typesense/issues/213

Could you tell me what version of Typesense you're seeing this on by doing a `GET /debug` curl request?

Also, if you're seeing this on Typesense Cloud, we'd need to upgrade your cluster to the latest public release. Could you email contact@typesense.org with the cluster ID and we can upgrade in-place for you.

**Comment by waako** [comment]
Thanks @jasonbosco!  
Bumping up local dev typesense docker image version fixed it alright, sorry I didn't find that issue before.

I've actually been discussing setup and touched on this issue with Christopher, so I think we are all a bit tired and having PEBCAK moments…
